help/user-guide/create-sample-isajson-compliant.md

```diff
@@ -0,0 +1,78 @@
+---
+title: SEEK User Guide - Create Sample in ISA-JSON compliant Experiments
+layout: page
+---
+
```

Comment:
I would add a first headline reading "Working with Samples in ISA-JSON compliant Experiments"